### PR TITLE
feat(test): add deis build integration test that supports v2-alpha

### PR DESCRIPTION
**Pull request by smothiki**
builds:list is working

### CODE REVIEW

**Comment by smothiki** [comment]
```
• Failure [14.017 seconds]
Builds
/Users/smothiki/deisworld/src/github.com/deis/workflow/_tests/builds_test.go:53
  with a logged-in user
  /Users/smothiki/deisworld/src/github.com/deis/workflow/_tests/builds_test.go:52
    with no app
    /Users/smothiki/deisworld/src/github.com/deis/workflow/_tests/builds_test.go:35
      can deploy the app [It]
      /Users/smothiki/deisworld/src/github.com/deis/workflow/_tests/builds_test.go:27

      Expected
          <string>: Creating build... done
          
      to contain substring
          <string>: Creating build... done
```

Got this error for builds:create which I don't understand

**Comment by helgi** [comment]
I believe others are getting similar oddness on Apps and Release

**Comment by smothiki** [comment]
referes to https://github.com/deis/workflow/issues/183

File: _tests/builds_test.go [hunk 1]
```diff
@@ -1,23 +1,53 @@
 package _tests_test
 
 import (
+	"fmt"
+
 	. "github.com/onsi/ginkgo"
-	// . "github.com/onsi/gomega"
```
Comment (bacongobbler) [inline _tests/builds_test.go]: can you remove this please?

File: _tests/builds_test.go [hunk 2]
```diff
@@ -1,23 +1,53 @@
...
+			})
 		})
 
-		It("can create a build from an existing image (\"deis pull\")", func() {
-			// "deis builds:create %s --app=%s", image, app
-			// curl app
-			// `deis pull %s -a %s --procfile="worker: while true; do echo hi; sleep 3; done"`, image, app
-			// "deis ps:scale worker=1"
-			// "deis logs --app=%s", app
+		Context("with a deployed app", func() {
```
Comment (bacongobbler) [inline _tests/builds_test.go]: should we remove this context if `deis builds` is already covered by "with no app"?

File: _tests/builds_test.go [hunk 3]
```diff
@@ -1,23 +1,53 @@
...
+				Expect(err).NotTo(HaveOccurred())
+				Expect(output).To(ContainSubstring(fmt.Sprintf("Creating Application... done, created %s", appName)))
+			})
+			It("can deploy the app", func() {
+				output, err := execute("deis builds:create %s -a %s", "deis/example-go", appName)
+				Expect(err).NotTo(HaveOccurred())
+				Expect(output).To(ContainSubstring("Creating build... done"))
+			})
+			It("can list app builds", func() {
+				// "deis builds:list --app=%s", app
```
Comment (arschles) [inline _tests/builds_test.go]: can you remove this comment

File: _tests/builds_test.go [hunk 4]
```diff
@@ -1,23 +1,53 @@
...
+					MatchRegexp(`[0-9a-f]{8}-([0-9a-f]{4}-){3}[0-9a-f]{12}`)))
+			})
 		})
 
-		It("can create a build from an existing image (\"deis pull\")", func() {
-			// "deis builds:create %s --app=%s", image, app
-			// curl app
-			// `deis pull %s -a %s --procfile="worker: while true; do echo hi; sleep 3; done"`, image, app
-			// "deis ps:scale worker=1"
-			// "deis logs --app=%s", app
```
Comment (arschles) [inline _tests/builds_test.go]: can you remove these comments

File: _tests/builds_test.go (hunk 2 above)
Comment (arschles) [inline _tests/builds_test.go]: the test cases are valid, and those in `with no app` seem mutually exclusive to these. IMO this context should be a sibling of its now-parent

Comment (smothiki) [inline _tests/builds_test.go]: As of now its better to leave the test empty as with an App deployed, we haven't covered any failure conditions.

**Comment by smothiki** [comment]
@arschles @bacongobbler PTAL

**Comment by arschles** [comment]
@smothiki there are still some comments from earlier that are un-addressed

**Comment by arschles** [comment]
@smothiki nevermind, GH is showing old diffs in the PR. I viewed the file directly and all looks good. Gave an LGTM

**Pull request merged**